### PR TITLE
fix: Add Rowland Adimoha profile

### PR DESCRIPTION
**Pull request by rowjay007**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/747"></a>

### CODE REVIEW

**Comment by rowjay007** [comment]
This is my first contribution the this react open source project

File: public/data/rowjay007.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Rowland Adimoha",
+  "bio": "Software developer | Open Sorcerer",
```
Comment (Panquesito7) [inline public/data/rowjay007.json:3]:
```suggestion
  "bio": "Software developer | Open Sourcerer",
```

File: public/data/rowjay007.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "avatar": "https://github.com/rowjay007.png",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/rowjay007",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me on Linkedin",
```
Comment (Panquesito7) [inline public/data/rowjay007.json:12]:
```suggestion
      "name": "Connect with me on LinkedIn",
```

File: public/data/rowjay007.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Rowland Adimoha",
+  "bio": "Software developer | Open Sorcerer",
+  "avatar": "https://github.com/rowjay007.png",
+  "links": [
+    {
+      "name": "Follow me on Github",
```
Comment (Panquesito7) [inline public/data/rowjay007.json:7]:
```suggestion
      "name": "Follow me on GitHub",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**